app/controllers/school_classes_controller.rb

```diff
@@ -0,0 +1,31 @@
+class SchoolClassesController < ActionController::Base
+  def new
+    @school_class = SchoolClass.new
+  end
+
+  def create
+    school_class = SchoolClass.create(school_class_params)
+    redirect_to school_class_path(school_class)
+  end
+  
+  def show
+    puts params
+    @school_class = SchoolClass.find(params[:id])
+  end
+
+  def edit
+    @school_class = SchoolClass.find(params[:id])  
+  end
+
+  def update
+    @school_class = SchoolClass.find(params[:id])
+    @school_class.update(school_class_params)
+    redirect_to school_class_path(@school_class)
+  end
```

Comment:
Since you're not using the `@school_class` instance variable in a view, it could be a regular variable.